### ISSUE TITLE
Set layer scale affects other layers too

### ISSUE THREAD

**Issue by Alaadel** [Scirra/Construct-bugs#7577]
## Problem description

I am trying to set the scale of a layer A, but the background is also scaled, even though the background has 0% scale rate. I have to set the background Parallax to 0%.
What if I need the background parallax to be 100% but I need to scale another layer at the same time?

## Attach a .c3p

[report-layer.zip](https://github.com/Scirra/Construct-bugs/files/13466023/report-layer.zip)

## Steps to reproduce

1. Preview
2. Change the Sliderbar
3. Go back to the editor
4. Set the bg layer Parallax to 0%,0%
5. Preview

## Observed result

after 2: the slider and bg scale with the ui layers
after 4: only ui is scaled (intended behavior)

## Expected result

only ui is scaled after step 2 and 4

**Comment by WilsonPercival** [comment]
@Alaadel I think it's similar to this: https://github.com/Scirra/Construct-bugs/issues/5886

The solution is to use `Unbounded scrolling`.

**Comment by WilsonPercival** [comment]
@Alaadel https://github.com/Scirra/Construct-bugs/issues/7146

**Comment by AshleyScirra** [comment]
This is indeed a duplicate of #7146. The problem is caused by scroll bounding, so you can check 'Unbounded scrolling' for the layout to avoid that.
